.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/GitVersionTfsTask/BuildScripts"]
```

Comment:
I'll focus on fixing everything on the task itself. Once we have something working and satisfying and depending on your preference I will either publish our tooling on npm or drop our tooling from the PR. Meanwhile it allows me to publish private extension for testing purpose more easily.